src/OpenFeature.Contrib.Hooks.Otel/MetricsConstants.cs

```diff
@@ -0,0 +1,23 @@
+namespace OpenFeature.Contrib.Hooks.Otel
+{
+    internal static class MetricsConstants
+    {
+        internal const string Unit = "double";
+
+        internal const string ActiveCountName = "feature_flag.evaluation_active_count";
+        internal const string RequestsTotalName = "feature_flag.evaluation_requests_total";
+        internal const string SuccessTotalName = "feature_flag.evaluation_success_total";
+        internal const string ErrorTotalName = "feature_flag.evaluation_error_total";
+
+        internal const string ActiveDescription = "active flag evaluations counter";
+        internal const string RequestsDescription = "feature flag evaluation request counter";
+        internal const string SuccessDescription = "feature flag evaluation success counter";
+        internal const string ErrorDescription = "feature flag evaluation error counter";
+
+        internal const string KeyAttr = "key";
```

Comment:
We should look into why this pkg doesn't provide the attributes for FF: https://github.com/open-telemetry/opentelemetry-dotnet/tree/af83eb1043a34f82ba3bfe20c8b674ac295cfb37/src/OpenTelemetry.SemanticConventions

Reply:
Not sure if they expect PRs for this or not, but we could open one.

Reply:
It seems the library has not "updated" since 2022: https://www.nuget.org/packages/OpenTelemetry.SemanticConventions/1.0.0-rc9.9

Might need some help to reopen engagement there